app/controllers/assignment_invitations_controller.rb

```diff
@@ -35,6 +35,8 @@ def setup; end
 
   # rubocop:disable MethodLength
   # rubocop:disable AbcSize
+  # rubocop:disable CyclomaticComplexity
+  # rubocop:disable PerceivedComplexity
```

Comment:
These are temporary disables. Once we rollout to 100% we can remove the old job and these disables.